zulip_bots/zulip_bots/bots/monkeytest/monkeytest.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any
+
+import requests
+
+
+class MonkeyTestHandler(object):
+    def initialize(self, bot_handler: Any):
+        self.config_info = bot_handler.get_config_info('monkeytest')
+
+    def usage(self):
+        return '''
+        This is bot for Monkeytest.it(site for testing sites)
+        '''
+
+    def handle_message(self, message, bot_handler):
+        content = message['content'].strip()
+        if content == 'help' or content == '':
+            bot_handler.send_reply(message,
+                                   'This is bot for Monkeytest.it(site for testing sites)\nWrite me adress of site you want to test')

```

Comment:
Once you address my comment about making `usage()` return a more detailed description, you could just use `self.usage()` here instead of hardcoding that message. :)